### ISSUE TITLE
<DropDown> component not read in Jaws (missing aria-live tag)

### ISSUE THREAD

**Issue by bedwar2** [CMSgov/design-system#1589]
**Describe the bug**
The DropDown Component does not contain the aria-live="assertive" which is a recommended fix for Jaws not reading a drop down box and it's options.

**To Reproduce**
1. Go to DropDown documentation at design.cms.gov
2. Inspect and see there is no "aria-live"
3. Use Jaws if you have it.  When tabbing to DropDown and using arrows, Jaws does not read the options

**Expected behavior**
Jaws should read the options.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome 
 - Version 98

**Smartphone (please complete the following information):**

**Additional context**
There may be a better way to address this, but 508 testers reporting they can't get Jaws to read the drop down box.   I had them run this command document.getElementsByTagName('select')[0].setAttribute('aria-live', 'assertive') and Jaws read the selections after that.

**Comment by zarahzachz** [comment]
Thanks for filing this bug! I've created an issue in [Jira](https://jira.cms.gov/browse/WNMGDS-1696) if you'd like to set yourself to watcher to monitor its progress.

**Comment by SparrowSpit** [comment]
@zarahzachz I see the Jira ticket is closed for this but this GH issue is open. Just thought I'd let you know.

**Comment by zarahzachz** [comment]
Closing this issue without a codefix.

The created [Jira ticket](https://jira.cms.gov/browse/WNMGDS-1696) was closed due to this bug being filed on an older implementation of Dropdown. A newer custom Dropdown has since been implemented and its markup differs greatly from the markup this issue tested against.
